doc/dev/crimson/crimson.rst

```diff
@@ -337,17 +334,17 @@ To use fio to test ``crimson-store-nbd``,
       [job0]
       offset=0
 
-#. Test the crimson object store using the fio compiled just now
+#. Test the Crimson object store using the custom ``fio`` built just now
```

Comment:
s/object store using/object store, using/

(Add a comma between the two clauses of the sentence.)